cookbook/security/voters_data_permission.rst

```diff
@@ -224,5 +228,8 @@ from the authorization checker is called.
 .. versionadded:: 2.6
     The ``security.authorization_checker`` service was introduced in Symfony 2.6. Prior
     to Symfony 2.6, you had to use the ``isGranted()`` method of the ``security.context`` service.
+    
+    ``denyAccessUnlessGranted()`` method was introduced in Symfony 2.6 as a shortcut.
```

Comment:
I remember having display issues on symfony.com with blank lines in versionadded directives. Can you please start a new one (repeat line 228 before this line)?